docs/src/architecture/state.md

```diff
@@ -1,74 +1,86 @@
 # State
+The state of the network describes the current condition of all individual accounts and note states. It describes what is currently the case. With its state model, using concurrent offchain state, Polygon Miden aims to realise private transactions, and execution and state bloat minimization. 
 
-The state of the network describes the current condition of all individual accounts and note states. It describes what is currently the case.
+Privacy mostly results from UTXO-like state, having notes and nullifiers, and zero-knowledge proofs. Execution bloat happens when transactions get re-executed by all participants of the network. State bloat describes the ever growing state stored in blockchain nodes. Zero-knowledge proofs can only contribute partly to the solution, a concurrent state model with offchain state is required.
```

Comment:
Privacy is realised from a UTXO-like state model consisting of notes and nullifiers combined with off-chain execution using zero-knowledge proofs. Execution bloat happens when transactions get re-executed by all participants of the network. State bloat describes the ever growing state stored in blockchain nodes.  Polygon Miden addresses these challenges via its state model that enables concurrent off-chain execution and off-chain storage.

Reply:
I like yours more, thanks